### PR TITLE
Adding a page about the Contributor License Agreement

### PR DESCRIPTION
**Pull request by jeffwilcox**
As we modernize and update the Microsoft Contributor License Agreement (CLA) experience, we are consolidating resources, and will be redirecting the `cla.opensource.microsoft.com` endpoint to `opensource.microsoft.com/cla` instead, to help inform people about the process, any specifics, the CCLA info, and also to point to a PDF of the CLA document.

This is preparing that work.

### CODE REVIEW

File: cla.html [hunk 1]
```diff
@@ -17,15 +17,101 @@ <h1 class="h2">Contributor License Agreements</h2>
...
+                    project:
+                </p>
+                 <ul>
+                    <li>
+                        When you open a pull request, the <code>license/cla</code> check is run by our 
+                        bot.
+                    </li>
+                    <li>
+                        If you've signed the CLA before, the status check goes green, and you're done, your
+                        pull request is ready to be reviewed and merged.
```
Comment (rimikab) [inline cla.html:52]: "If you've signed the CLA before, the status check goes green, and you're **done. Your** pull request is ready to be reviewed and merged."

File: cla.html [hunk 2]
```diff
@@ -17,15 +17,101 @@ <h1 class="h2">Contributor License Agreements</h2>
...
+                    </li>
+                    <li>
+                        If you have not signed the CLA, the pull request will receive a comment annotated 
+                        with the agreement and more information about signing the agreement. You will reply
+                        with a comment if you accept the agreement yourself, or, if you've been authorized to
+                        sign it on behalf of your company, there is syntax for that as well.
+                    </li>
```
Comment (rimikab) [inline cla.html:59]: "If you have not signed the CLA **at any point before**, the pull request will receive a comment annotated with the agreement and more information about signing the agreement. You will reply with a comment **to accept the agreement on behalf of yourself**, or, if you've been authorized to sign it on behalf of your company, **there is a different syntax for that."**

File: cla.html [hunk 3]
```diff
@@ -17,15 +17,101 @@ <h1 class="h2">Contributor License Agreements</h2>
...
+                    <li>
+                        After signing, the status check will pass and you're all set. <strong>Thanks for your contribution!</strong>
+                    </li>
+                    <li>
+                        <em>NOTE:</em><br />
+                        If you're an active Microsoft employee or vendor, and have a linked GitHub account,
+                        the check will also go green. Microsoft employees can learn more about linking at 
+                        <a href="https://aka.ms/opensource" target="_new">aka.ms/opensource</a>.
+                    </li>
```
Comment (rimikab) [inline cla.html:68]: A bit confused here - learn about linking what?

**Review by rimikab — CHANGES_REQUESTED** [review]
just minor grammar

File: cla.html (hunk 3 above)
Comment (jeffwilcox) [reply cla.html:68]: their GitHub account

File: cla.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            <div class="col-md-10 col-lg-7 mx-auto">
+                <h1 class="h2">Contributor License Agreements</h2>
+            </div>
+        </div>
+    </div>
+    <div class="wrapper-full bg-light">
+        <div class="wrapper my-6 py-4">
+            <div class="col-md-10 col-lg-7 mx-auto">
+                <h3 class="h4 font-weight-400 mb-4">
+                  Microsoft projects have a Contributor License Agreement (CLA). The Microsoft
```
[the remark this answers is not in the document]
Comment (jeffwilcox) [reply cla.html]: I can't find this text to update?

[the remark this answers is not in the document]
Comment (tellarin) [reply cla.html]: This was removed on your second commit along with the mention of the CLA only needing to be signed once. So I guess it can be dismissed.

**Review by tellarin — APPROVED** [review]
LGTM.

**Pull request merged**